tests/acceptance/ape-config.yaml

```diff
@@ -5,8 +5,8 @@ plugins:
 
 dependencies:
   - name: nucypher-contracts
-    github: nucypher/nucypher-contracts
-    ref: main
+    github: derekpierre/nucypher-contracts
```

Comment:
Yes. It can be merged with that reference since the PR is going into an EPIC (#3226), but it will be changed to `nucypher-contracts/main` as part of merging the epic.